app/llama_sensei/backend/add_courses/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM pytorch/pytorch:2.3.1-cuda11.8-cudnn8-devel
+
+RUN mkdir /app
+COPY . /app/
+
+WORKDIR /app
+RUN apt-get update && \
+    apt-get install -y git
```

Comment:
to clean up the os, consider to add this

```dockerfile
  apt-get clean && \
  rm -rf /var/lib/apt/lists/*
```

Reply:
Fixed in last commit